sites/funmooc/src/backend/base/tests/test_views.py

```diff
@@ -49,13 +93,21 @@ def test_views_redirect_edx_courses_fallback_search_page(self):
         nor the organization page can be found.
         """
         PageFactory(
-            reverse_id="search",
+            reverse_id="courses",
```

Comment:
I don't mean there's a mistake here, but I don't get the relation between search and this reverse_id.

Reply:
Because, on fun-mooc.fr, the reverse_id of the search page is `courses` not `search`.